xm-commons-lep-groovy/src/main/java/com/icthh/xm/commons/lep/LepPathsUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.icthh.xm.commons.lep;
+
+import com.icthh.xm.commons.config.client.service.TenantAliasService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LepPathsUtils {
+
+    public static List<String> getLepBasePaths(String tenant, String appName, TenantAliasService tenantAliasService) {
+        List<String> paths = new ArrayList<>(List.of(
+            tenant + "/" + appName + "/lep",
+            tenant + "/commons/lep",
```

Comment:
I saw common related constant in `GroovyLepEngine` do we need to use common constant?